app/views/certificates/show.html.slim

```diff
@@ -34,9 +34,11 @@
           span.kind-list
             input*{ type: 'checkbox', checked: value } onclick='return false;'
             = "#{t("assignment_kinds.#{kind}")}"
+        input*{ type: 'checkbox', checked: "#{@certificate.group_offer}" } onclick='return false;'
```

Comment:
I think you can simply do `input type="checkbox" checked="#{@certificate.group_offer}" onclick="return false"`, or use the `check_box_tag` helper